manifests/vanilla/vsphere-csi-driver.yaml

```diff
@@ -162,6 +162,8 @@ data:
   "max-pvscsi-targets-per-vm": "true"
   "multi-vcenter-csi-topology": "true"
   "csi-internal-generated-cluster-id": "true"
+  "csi-vc-sessions-auto-disconnect": "true"
```

Comment:
we can not enable new feature right before validation

Reply:
also this code change is applicable to both supervisor and vanilla so curious why we have this gate only for vanilla.

Can we not take this as bug fix rather than new feature?

Reply:
will take this as bug fix, removed feature gate